### DISCUSSION TITLE
Weird Text Behaviour

### DISCUSSION THREAD

**Discussion by chxp82q** [wezterm/wezterm#672] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33443763/114267765-728cf180-99b2-11eb-87ea-170485563d83.png)

After building to the latest commit, I'm getting this weird behavior where some text and glyphs don't show. For now, I'll roll back to stable release.

Thanks!

**Comment by wez** [comment, marked as answer]
Should be fixed in c31298364381de9c9a4faf1e914c5ec7e810e2eb
